.github/workflows/deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - "sprint-**-release"
```

Comment:
Maybe execute this on all releases? When we have an out-of band release, this will not get destroyed

Reply:
yes was thinking about that too, so we also deploy for release candidates. which saves time in the scrum meeting